suiteData.js

```diff
@@ -12,8 +15,14 @@ class SuiteData {
             throw new Error("A suite 'name' parameter should be included on the query string or in the request body.");
         }
         var tests = obj && obj.tests;
+        if (_.isString(tests) && fs.existsSync(sanitize(tests))) {
```

Comment:
you can't use the sync version of the fs functions...
you can use util.promisify to convert the existing callback based functions to promised functions